### ISSUE TITLE
Unexpected 'valid-v-bind' error for bind parameter when value starts with hashtag

### ISSUE THREAD

**Issue by cristianomoraes** [vuejs/eslint-plugin-vue#1052]
**Tell us about your environment**	
* **ESLint version:**	5.16.0
* **eslint-plugin-vue version:**	5.2.3
* **Node version:**	12.3.1

**Please show your full configuration:**

```json
module.exports = {
    root: true,

    parserOptions: {
        parser: "babel-eslint",
        sourceType: "module",
    },

    env: {
        browser: true,
    },

stricter rules.
    extends: ["plugin:vue/essential", "@vue/prettier", "plugin:prettier/recommended"],

    plugins: ["vue", "prettier"],

    globals: {
        ga: true, 
        cordova: true,
        __statics: true,
        env: true,
        process: true,
    },

    rules: {
        "prettier/prettier": "error",
        "prefer-promise-reject-errors": "off",
        "no-unused-vars": ["warn", { args: "after-used" }],

        "no-console": process.env.NODE_ENV === "production" ? "error" : "off",

        "no-debugger": process.env.NODE_ENV === "production" ? "error" : "off",
    },
}
```

**What did you do?**	

```vue
                    <croppa
                        v-model="croppa"
                        :width="200"
                        :height="200"
                        placeholder="Clique para selecionar uma imagem"
                        placeholder-color="#000"
                        :placeholder-font-size="16"
                        :show-remove-button="true"
                        remove-button-color="black"
                        :show-loading="true"
                        :loading-color="#606060"
                        :loading-size="50"
                    />
```

**What did you expect to happen?**
Get no errors

**What actually happened?**
I got the following

  20:25  error  'v-bind' directives require an attribute value                                        vue/valid-v-bind
  20:41  error  Parsing error: Line 1: Unexpected digit after hash token

> 1 | 0(#606060)
    |   ^  vue/no-parsing-error

✖ 2 problems (2 errors, 0 warnings)

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Comment by ota-meshi** [comment]
Hi @cristianomoraes.

I think you need to remove the colon.

```diff
-                        :loading-color="#606060"
+                        loading-color="#606060"
```

**Comment by cristianomoraes** [comment]
Damn.. of course! Based at an example of this component I assumed it _had_ to be bound what if I had thought through for a half minute doesn't make any sense! That was a painful stupid mistake of me. Thank you.
